### PR TITLE
Feature/update variable estimates

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: src/FactorGraph01.jl [hunk 1]
```diff
@@ -737,6 +737,8 @@ function doautoinit!(dfg::T,
         end
         pts,inferdim = predictbelief(dfg, vsym, useinitfct, logger=logger)
         setValKDE!(xi, pts, true, inferdim)
+        #TODO test
+        setVariablePosteriorEstimates!(xi)
```
Comment (Affie) [inline src/FactorGraph01.jl]: @dehann, I think adding extra data breaks compare test, but I haven't had time to dig into the details.

Comment (dehann) [reply src/FactorGraph01.jl]: these compare functions recently moved to DFG, so we should just tie up there too -- thanks!

**Comment by dehann** [comment]
Hi @GearsAD , just mention you to keep mental note prior to DFG v0.5.0 -- that we may need to modify some of the compare stuff either in IIF or DFG when adding this PR.  I think this PR should land in the IIF v0.8.0 release, unless there are any objections.

**Comment by Affie** [comment]
Lets hold off on this one and do it all in one go pending https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/147#issuecomment-542091523 I think we are close enough to a final decision.
I use it currently, but I'm happy working on a branch for now.

**Comment by Affie** [comment]
needs https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/pull/183

**Comment by dehann** [comment]
addresses #214

**Pull request merged**